src/core/client/stream/tabs/Comments/Stream/PostCommentForm/CreateCommentMutation.ts

```diff
@@ -406,11 +418,31 @@ export const CreateCommentMutation = createMutation(
             if (expectPremoderation) {
               return;
             }
-            sharedUpdater(environment, store, { ...input, tag }, uuidGenerator);
+
+            if (tag) {
+              sharedUpdater(
+                environment,
+                store,
+                { ...input, tag },
+                uuidGenerator
+              );
+            } else {
+              sharedUpdater(environment, store, input, uuidGenerator);
+            }
+
             store.get(id)!.setValue(true, "pending");
           },
           updater: (store) => {
-            sharedUpdater(environment, store, { ...input, tag }, uuidGenerator);
+            if (tag) {
+              sharedUpdater(
+                environment,
+                store,
+                { ...input, tag },
+                uuidGenerator
+              );
+            } else {
+              sharedUpdater(environment, store, input, uuidGenerator);
+            }
```

Comment:
If we remove `tag` from `input` then we don't need this change here anymore.

Reply:
To be correct, the `sharedUpdater` should actually use the `tags` that comes from the `response` instead.